index.html

```diff
@@ -3409,6 +3409,44 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+      <section>
+        <h3>Media Types</h3>
+
+        <p>
+          Media types as defined in [[RFC6838]] serve a useful purpose with verifiable 
+          credentials, especially in regards to identifcation of a particular syntax 
+          that is in use with a verifiable credential.
+        </p>
+        <p>
+          Syntaxes SHOULD be identified by a media type, and certain conventions as 
+          outlined in this section should be followed when defining or using media types 
+          with verifiable credentials.
+        </p>
+        <p>
+          Any media types associated with the core data model are listed in the section on
+          <a href="#iana-considerations">IANA Considerations</a> for registration with IANA.
+        </p>
+        <p>
+          When defining a media type for use with verifiable credentials, for instance in a 
+          specification that defines a specific syntax, use of the term `verifiable` implies
+          the presence of a <a href="#proofs-signatures">proof</a> with the credential.  
+          This is also the case if the term is abbreviated, such as in `vc+ld+json`. 
```

Comment:
We might want a ref to the JWT spec here to be clear on what this abbreviation language is about.

```suggestion
          This is also the case if the term is abbreviated in accordance with other specs, such as in `vc+ld+json`. 
```